src/clib/lib/job_queue/torque_driver.cpp

```diff
@@ -671,6 +671,7 @@ void *torque_driver_submit_job(void *_driver, std::string submit_cmd,
 static job_status_type
 torque_driver_get_qstat_status(torque_driver_type *driver,
                                const char *jobnr_char) {
+    1/0
```

Comment:
This is not an exception but it might raise `SIGFPE` POSIX signal. Division by zero is undefined behaviour (UB), so "anything" can happen. In practice this is likely to be completely optimised out, so you don't get any error.

See: https://godbolt.org/z/n1zfxWEcd and notice how the compiler optimises this to `xor eax, eax; ret`, which just means `return 0` in assembly.

If you want to check that exceptions are caught, use eg. `throw py::runtime_error("Hello from C++");`